cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/hdfs/AbstractHDFSStats.java

```diff
@@ -43,14 +43,17 @@
 import java.net.URI;
 import java.net.URL;
 import java.util.Collection;
+import java.util.concurrent.TimeUnit;
 import javax.annotation.Nullable;
 
 /**
  * Base class for {@link OperationalStats} for HDFS
  */
 public abstract class AbstractHDFSStats extends AbstractOperationalStats {
   private static final Logger LOG = LoggerFactory.getLogger(AbstractHDFSStats.class);
-  private static final Logger READ_FAILURE_LOG = Loggers.sampling(LOG, LogSamplers.limitRate(60000));
+  private static final Logger READ_FAILURE_LOG =
```

Comment:
This one logs for all failures, not just HDFS and also it does not log the stack trace.